packages/fxa-shared/subscriptions/configuration/base.ts

```diff
@@ -96,24 +96,29 @@ export interface BaseConfig {
     };
   };
   support?: SupportConfig;
+  promotionCodes?: string[];
 }
 
-export const baseConfigSchema = joi
-  .object({
+export const minimalConfigSchema = joi.object({
+  id: joi.string().optional().allow(null).allow(''),
+  urls: urlsSchema,
+  uiContent: uiContentSchema,
+  styles: stylesSchema,
+  locales: joi.object({}).pattern(
+    joi.string(),
+    joi.object({
+      uiContent: uiContentSchema,
+      urls: urlsSchema,
+      support: supportSchema,
+    })
+  ),
+  support: supportSchema,
+});
+
+export const baseConfigSchema = minimalConfigSchema
+  .keys({
```

Comment:
Should the optional `productSet` be in here as well, since both products and plans can have them?

Reply:
Yep. 👍